config/CMakeLists.txt

```diff
@@ -16,6 +16,13 @@ set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH}" PARENT_SCOPE)
 # Check for available features
 # Note: users can overwrite the automatic check by setting the value at configure time
 include(CheckFortranSourceRuns)
+if (NOT DEFINED WITH_CBOOL)
+  check_fortran_source_runs(
+    "use, intrinsic :: iso_c_binding, only: c_bool; integer, parameter :: lk = kind(.true.); if (c_bool == lk) stop 1; end"
```

Comment:
This can fail due to line-length, better make it a multiline string.

Reply:
done